subtrack.DAL/Entities/Subscription.cs

```diff
@@ -18,4 +18,8 @@ public class Subscription
     public int FirstPaymentDay { get; set; } 
     [Required]
     public DateTime LastPayment { get; set; }
+
+    public BillingOccurrence BillingOccurrence { get; set; }
+
+    public int BillingInterval { get; set; }
```

Comment:
Add the data annotation mentioned in the Acceptance Criterias. Might require you to recreate the migration. Just remove the new migration file, your database and then create a new migration

Reply:
Will do. I can add a new migration that handles this case rather than deleting and recreating the existing one.

Reply:
> Will do. I can add a new migration that handles this case rather than deleting and recreating the existing one.

That works aswell but imo its better to have one migration, less clutter

Reply:
`Range` attribute isn't adding anything to the migrations. Am i missing something here or is the range attribute not used by EF?

Reply:
> `Range` attribute isn't adding anything to the migrations. Am i missing something here or is the range attribute not used by EF?

It doesnt need to be enforced in the database. It will be used by the data annotations validator on the upsert page